src/host/toolsResourceLoader.ts

```diff
@@ -3,6 +3,9 @@
 
 import { encodeMessageForChannel } from "../common/webviewEvents";
 
+export interface IRoot {

```

Comment:
Let's move this into `host.ts` and just use it from there since it is the only thing that really needs it.